MicrowaveGame/Assets/Resources/Scenes/Gameplay.unity

```diff
@@ -500,6 +500,10 @@ PrefabInstance:
       propertyPath: m_IsTrigger
       value: 0
       objectReference: {fileID: 0}
+    - target: {fileID: 8573620642782398239, guid: 71a2878986dd1e048b0d34eac7a690b9, type: 3}
+      propertyPath: AimAssistConeAngle
+      value: 0
+      objectReference: {fileID: 0}
```

Comment:
I don't think this was intentional but it looks like the aim assist cone angle has been set to 0 for the gameplay scene.